docs/detections/rules-prebuilt.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[rules-prebuilt]]
+== Install Elastic prebuilt rules
+
+:frontmatter-description: Start detections quickly with prebuilt rules designed and updated by Elastic.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+Follow these guidelines to start using the {es-sec-app}'s <<prebuilt-rules, prebuilt rules>>, keep them updated, and make sure they have the data needed to run successfully. 
```

Comment:
It's a lot easier to change the name of this page, from `rules-prebuilt` to `prebuilt-rules-management`, because it's a net-new page created in this PR. No established SEO or redirects we have to consider.

Renaming the existing `Prebuilt rule reference` page is trickier since it's such a long-established URL with lots of SEO history, AND there might be other internal processes that depend on that location (TRADE manages publishing of prebuilt rule docs, and I'm not familiar enough to know how this would affect their process). But I think it's worth investigating for the reasons you bring up. I've created a separate issue (#3597)  so we can follow up on renaming that page while also moving forward with the rest of this PR.

Reply:
Thank you @joepeeples, especially for such a thorough comment with all the links. Really appreciate this!